### ISSUE TITLE
Document Active Directory integration via OIDC

### ISSUE THREAD

**Issue by tomponline** [canonical/lxd#13310]
We've been advised by the Canonical Identity team that it is possible to use OIDC with Microsoft Active Directory.

As this is commonly something that is asked for by users, it would be useful to confirm and document this process.

**Comment by tomponline** [comment]
@markylaing did this get done?

**Comment by markylaing** [comment]
No. I wasn't aware of this issue. We could add something similar to the Entra ID docs which use Keycloak, since Keycloak supports it. I wouldn't want to recommend ADFS anywhere in the docs.
